src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebeforecasedefaultcolon/Example1.java

```diff
@@ -12,9 +12,9 @@
 
 import java.time.DayOfWeek;
 
+// xdoc section -- start
 class Example1 {
   void example() {
-    // xdoc section -- start
     switch(1) {
       case 1 : // violation '':' is preceded with whitespace'
```

Comment:
I was getting checkstyle errors that `<source>` examples are indented. I am moving delimiters to include class definitions too. This is what was decided at some point after those examples were merged anyways.

Though the issue still stays. I can copy functionality from `SnippetMacro` https://github.com/apache/maven-doxia/blob/master/doxia-core/src/main/java/org/apache/maven/doxia/macro/snippet/SnippetReader.java. This is how they deal with indentation.

Reply:
Moved to https://github.com/checkstyle/checkstyle/pull/13394